### PR TITLE
PB-1187 added config.videoQuality.restrictVideoHeightByParticipants

### PR DESCRIPTION
**Pull request by segnord**
in custom-config.js add the following to limit video height by participants

```javascript
config.videoQuality = {
    restrictVideoHeightByParticipants: {
        10: 360,
        20: 180
    }
}
```

### CODE REVIEW

**Comment by skolmer** [comment]
Only height not width?

**Comment by segnord** [comment]
Yes, basically that's how quality is controlled across the whole application.

**Comment by segnord** [comment]
Can't go below 180

File: react/features/video-quality/reducer.js [hunk 1]
```diff
@@ -69,9 +69,29 @@ function _setConfig(state, { config }) {
...
+    const heightByParticipant = config?.videoQuality?.restrictVideoHeightByParticipants;
+    const heightByParticipantMap = validateRestrictVideoHeightByParticipants(heightByParticipant);
+
     if (configuredMap && !convertedMap) {
         logger.error('Invalid config value videoQuality.minHeightForQualityLvl');
     }
 
-    return convertedMap ? set(state, 'minHeightForQualityLvl', convertedMap) : state;
+    logger.error(heightByParticipant);
+    logger.error(heightByParticipantMap);
```
Comment (skolmer) [inline react/features/video-quality/reducer.js:80]: looks like these log outputs got here by accident

Comment (segnord) [reply react/features/video-quality/reducer.js:80]: yes, those need to be removed

**Pull request closed without merging**